recipes/dioptas/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "Dioptas" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "5911464f2141f19720d8b987796d86344602737c9e4c34309d0b0dfd2b32d90b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Dioptas/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py build install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - pyqt
+    - pyqtgraph
+    - lmfit
+    - fabio
+    - pyfai
+    - pycifrw
+    - future
+    - scikit-image
+    - qtpy
+
+  run:
+    - python
+    - setuptools
```

Comment:
Do you need ``setuptools`` after the build?

Reply:
nope, deleted it